update_translations.sh

```diff
@@ -0,0 +1,5 @@
+pyside-lupdate -verbose qfield_sync_py.pro # this parses the py files but not the ui files
```

Comment:
whoops, this file should not be here

On Sun, Jun 26, 2016 at 9:42 PM, Matthias Kuhn <notifications@github.com>
wrote:

> In update_translations.sh
> <https://github.com/opengisch/QFieldSync/pull/1#discussion_r68508363>:
>
> > @@ -0,0 +1,5 @@
> > +pyside-lupdate -verbose qfield_sync_py.pro # this parses the py files but not the ui files
>
> why pyside? I think this shouldn't be a dependency (we use pyqt).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opengisch/QFieldSync/pull/1/files/f7003aea582a577830a49dc0a63497ad050eaa6c..7f35c9db2ac6899299498142c2a3985af20f9a98#r68508363>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA7VK10U25dKmRI4kQeGjXtGEXsBCtpgks5qPtY5gaJpZM4IpZ7z>
> .
>